eng/testing/xunit/vstest.xunit.json

```diff
@@ -0,0 +1,5 @@
+{
+    "diagnosticMessages": false,
+    "longRunningTestSeconds": 120,
```

Comment:
So this support is now built-in in VSTest?

Reply:
Yeah but I still need to play around with it. There's a blame mode which creates a sequence.xml which tells you which tests have run and which haven't but I don't know if that mode slows down the test invocation. cc @nohwnd